app/Jobs/Curations/AddStatus.php

```diff
@@ -45,24 +46,22 @@ public function __construct(Curation $curation, CurationStatus $curationStatus,
      */
     public function handle()
     {
-        if ($this->hasCurationCurationStatus() && ($this->isCurrentStatus() || $this->isPreviousDatedStatus())) {
-            return;
-        }
+        if($this->hasCurationCurationStatus() && $this->isCurrentStatus()) { return; }
+        if($this->isSameStatusOnDate()) { return; }
```

Comment:
I don't actually think we want to prevent same status on date-- just not consecutive status. For instance, it's OK to go to published, then figure out there was an issue, go to revised, then back to published on the same day.